packages/apex-node/src/tests/types.ts

```diff
@@ -49,6 +49,10 @@ export type AsyncTestConfiguration = {
    */
   testLevel: TestLevel;
   skipCodeCoverage?: boolean;
+  /**
+   * Does not wait for test run to complete and returns the test run id immediately
+   */
+  exitEarly?: boolean;
```

Comment:
Can we use a more descriptive name? like `waitForTestRunToComplete` or `exitOnTestRunIdAcquired`